packages/eslint-config-base/package.json

```diff
@@ -11,18 +11,18 @@
     "type": "git",
     "url": "git+https://github.com/LabKey/labkey-ui-components.git"
   },
-  "author": "kevink@labkey.com",
+  "author": "LabKey",
   "license": "Apache-2.0",
   "dependencies": {
-    "@typescript-eslint/eslint-plugin": "5.27.0",
-    "@typescript-eslint/parser": "5.27.0",
-    "eslint": "8.16.0",
-    "eslint-config-prettier": "8.5.0",
-    "eslint-plugin-import": "2.26.0",
-    "eslint-plugin-jest": "26.4.6",
-    "eslint-plugin-only-warn": "1.0.3",
-    "eslint-plugin-prettier": "4.0.0",
-    "eslint-plugin-typescript-sort-keys": "2.1.0",
-    "prettier": "2.6.2"
+    "@typescript-eslint/eslint-plugin": "6.7.0",
+    "@typescript-eslint/parser": "6.7.0",
+    "eslint": "8.49.0",
```

Comment:
One issue I'm seeing that I'd appreciate someone's assistance with is a deprecation warning from ESLint in IntelliJ:

<img width="774" alt="Screenshot 2023-09-20 at 9 02 16 AM" src="https://github.com/LabKey/labkey-ui-components/assets/3926239/85e58b5b-dfc2-4de2-95e5-6fcf87f3593e">

I believe this is coming from some base configuration we're relying on but haven't tracked it down yet.

Reply:
Ugh. https://stackoverflow.com/questions/76769634/why-am-i-getting-a-deprecationwarning-originalkeywordkind-deprecated-error

Reply:
I've addressed this by disabling the eslint plugins `jest` and `typescript-sorted-keys`. They both share an indirect dependency on an outdated package, tsutils, which will need to be removed on their side.